packages/auditd/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.18.0"
+  changes:
+    - description: Handle ENRICHED audit formatr.
```

extra r after format?

```suggestion
    - description: Handle ENRICHED audit format.
```

---

Thanks